pytorch/training/docker/1.5.0/py3/Dockerfile.cpu

```diff
@@ -106,6 +106,16 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
  && pip install --no-deps --no-cache-dir -U \
     https://torchvision-build.s3.amazonaws.com/1.5.0/cpu/torchvision-0.6.0-cp36-cp36m-linux_x86_64.whl
 
+# Adding support to SageMaker for MPI/Horovod
```

Comment:
What about 1.5.1 ?

Reply:
Rest of work here is dependent on SageMaker ML Frameworks Team changes, the latest they support as of now is 1.5.0 (https://github.com/aws/sagemaker-pytorch-training-toolkit/tree/master/docker).